### ISSUE TITLE
Should all meetups share same branding?

### ISSUE THREAD

**Issue by praveenpuglia** [VueJS-IN/india.vue.community#4]
My idea with india.vue.community is for all the smaller Vue communities in India to come under one umbrella. People shouldn't need to go multiple different places to find that information.

Branding will play a key role in that I belive.

What if all the cities shared similar branding like
- Bangalore : [Vue India Logo] Bangalore
- Hyderabad : [Vue India Logo] Hyderabad 

## Quick Mockups!(With an icon I proposed as new logo entry)
<img width="639" alt="image" src="https://user-images.githubusercontent.com/313363/71058805-fa7c3600-2186-11ea-86ca-94ae1179cf44.png">
<img width="608" alt="image" src="https://user-images.githubusercontent.com/313363/71058825-0a941580-2187-11ea-92ba-e31b6d0e2474.png">

This also makes the graphics making part for the entire community easy.

**Comment by chintanbanugaria** [comment]
Makes sense. However, this might loose the individual identity of the meet ups. I could be wrong though.

**Comment by praveenpuglia** [comment]
I totally get that point and agree to that. Depends on how we want to project ourselves as one entity with multiple units or multiple entities with one common ground.
